mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -131,3 +125,103 @@ These two unsupported fields were introduced incorrectly. They are now removed.
 ### `liveness_probe.tcp_socket` is now removed
 
 This unsupported field was introduced incorrectly. It is now removed.
+
+## Resource: `google_firebase_project_location`
+
+### `google_firebase_project_location` is now removed
+
+In `4.X`, `google_firebase_project_location` would implicitly trigger creation of an App Engine application with a default Cloud Storage bucket and Firestore database, located in the specified `location_id`. With `5.0.0`, these resources should instead be set up explicitly.
+
+- If you only used `google_firebase_project_location` to create a default Storage bucket, 
```

Comment:
I think that it would probably be fine to remove the content from here to the `Upgrade instructions` header - this was previously the "Detailed changes section. There aren't really any changes to the resources, just removing the implicit creation, and the information covered here is better covered in the Upgrade Instructions section.

If there's anything important that you think would be lost, could it be integrated into the Upgrade Instructions section instead?

Reply:
My understanding of the `Upgrade instructions` section is how to edit an existing terraform config to be usable in 5.x.x. However, the list here is more geared towards how to create new resources going forward without an existing/old resource.

Reply:
Hm, I see. I'd prefer to keep the upgrade guide focused on the upgrade steps, but giving guidance on how to create new resources going forward makes sense. What would you think about moving the "For more information on configuring Firebase resources with Terraform" line up to this point to cover the "new resources going forward" information instead of writing out this list?

Reply:
Good idea. How about now?